javascript-practice/src/js/helpers/helpers.js

```diff
@@ -0,0 +1,29 @@
+import { TIME_OUT_SEC, TIME_OUT_ERROR, regex } from '../constants/constant';
+import FirebaseService from '../services/firebaseService';
+
+export const validatePassword = (password) => {
+  return regex.test(password);
+};
+
+export const timeout = (s) => {
+  return new Promise((reject) => {
+    setTimeout(() => {
+      reject(TIME_OUT_ERROR);
+    }, s * 1000);
+  });
+};
+
+export const createIdUser = () => {
```

Comment:
Your database should do this, if not, it should be in model

Reply:
Fixed.